CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Disabled caching for webviews.
 * Expanded the flag (enablePreviousNameRecording) to add current view name as segmentation to custom events. (Experimental!)
 
+* Mitigated an issue where session was starting after without merge while app was in background.
```

Comment:
'Could', not 'was'. Could indicates 'it could have happened'. 'was' indicates 'it was happening each time'